src/mono/msbuild/AndroidTestRunner/AndroidTestRunner.cs

```diff
@@ -63,7 +63,7 @@ protected override IEnumerable<TestAssemblyInfo> GetTestAssemblies()
 
     protected override TestRunnerType TestRunner => TestRunnerType.Xunit;
 
-    protected override string? IgnoreFilesDirectory => null;
+    protected override string? IgnoreFilesDirectory => Environment.CurrentDirectory;
```

Comment:
PR looks good, but since I added the property, we can use it and not deal with the dir, the properties are virtual now and return a null, so we should be ok.

Reply:
I don't see it in the latest version of xharness yet.

Reply:
Just got merged. Give the CI some time.

Reply:
https://github.com/dotnet/runtime/pull/35662 has the bump